Plugins/Flow.Launcher.Plugin.Explorer/Settings.cs

```diff
@@ -14,7 +14,7 @@ public class Settings
         // as at v1.7.0 this is to maintain backwards compatibility, need to be removed afterwards.
         public List<AccessLink> QuickFolderAccessLinks { get; set; } = new List<AccessLink>();
 
-        public bool UseWindowsIndexForDirectorySearch { get; set; } = true;
+        public bool UseWindowsIndexForDirectorySearch { get; set; } = false;
```

Comment:
Have we already checked this condition?

Reply:
Sorry what do you mean check?

Reply:
Oh yeah it's used, when I wrote the plugin I had already considered this would be something we need to expose to the ui, so it's already in use and I didn't have to add any for this change.